routes/htmlRoutes.js

```diff
@@ -1,23 +1,13 @@
-var db = require("../models");
+const { SurveyResults } = require("../models");
 
 module.exports = function(app) {
   // Load index page
   //Note: what are we going to render from the database on the index page? Maybe not all the data, no? Maybe carousel of most recent tests?
   app.get("/", function(req, res) {
-    db.Example.findAll({}).then(function(dbExamples) {
+    SurveyResults.findAll({}).then(data => {
       res.render("index", {
         msg: "Free Personality Test",
-        examples: dbExamples
-      });
-    });
-  });
-
-  // Load example page and pass in an example by id
-  // Notes: not sure what this is doing, and if we can delete?
-  app.get("/example/:id", (req, res) => {
```

Comment:
This just gets the first promise that gets returned from the query and displays it to the page